Source/WebCore/accessibility/AccessibilityObject.cpp

```diff
@@ -2711,8 +2711,8 @@ AccessibilityRole AccessibilityObject::ariaRoleToWebCoreRole(const String& value
 {
     if (value.isNull() || value.isEmpty())
         return AccessibilityRole::Unknown;
-
-    for (auto roleName : StringView(value).split(' ')) {
+    String valueWithSimplifiedWhiteSpace = value.simplifyWhiteSpace(isASCIIWhitespace);
```

Comment:
I think you can use `auto` instead of specifying `String` here. Maybe we can shorten the variable name to something like `simplifiedValue`?

Reply:
Thank you for your review.
I applied your suggestion.